spec/requests/api/v3/providers/suggest_provider_spec.rb

```diff
@@ -0,0 +1,90 @@
+require "rails_helper"
+
+describe "GET /provider-suggestions" do
+  let(:jsonapi_renderer) { JSONAPI::Serializable::Renderer.new }
+  let(:organisation) { create(:organisation) }
+  let(:provider1) { create(:provider, provider_name: "PROVIDER 1", organisations: [organisation]) }
+  let(:provider2) { create(:provider, provider_name: "PROVIDER 2", organisations: [organisation]) }
+  let(:next_recruitment_cycle) { find_or_create(:recruitment_cycle, :next) }
+  let(:provider3) { create(:provider, provider_name: provider2.provider_name, organisations: [organisation], recruitment_cycle: next_recruitment_cycle) }
+  let(:provider4) { create(:provider, organisations: [organisation], recruitment_cycle: next_recruitment_cycle) }
+
+  before do
+    provider1
```

Comment:
This is a small thing in this file but adds up over the test suite. We should try and limit the amount of resources we create in the tests.

Reply:
@gpeng - yep good call. I've refactored the spec